en/core-libraries/components/authentication.rst

```diff
@@ -195,12 +195,7 @@ working with a login form could look like::
                 // Prior to 2.3 use
                 // `return $this->redirect($this->Auth->redirect());`
             }
-            $this->Session->setFlash(
-                __('Username or password is incorrect'),
-                'default',
-                array(),
-                'auth'
-            );
+            $this->Flash->error(__('Username or password is incorrect'));
```

Comment:
I would include the 'old' way to do this here as well. The docs don't only cover 2.7+.